README.md

```diff
@@ -152,6 +152,26 @@ Do this if you want to use the remote server for building and deploying.
 
     Where the `$user` shell variable contains the name of your shell account on the remote machine.
 
+### Configuring `geth-<testnet/mainnet>-vm` machine
+This step configure geth mainnet or geth testnet instance on the remote server.
```

Comment:
Please reword:

``` markdown
### Setting up Ethereum client on a separate machine
This step installs and configures Geth on a separate machine in Google Compute Engine.
This is optional since Geth be deployed automatically as a part of a Concent cluster but when you have multiple clusters, having one shared instance of the client allows you to use less resources.
Concent does not use Geth very heavily so one instance of the blockchain client can handle many Concent instances.
It also makes deployment faster and more reliable since it's not necessary to wait for Geth synchronization every time it's updated or redeployed.

This machine comes in two flavors: `geth-testnet` and `geth-mainnet`.
If you're running both test clusters and a production cluster, you need both.
```

Reply:
Sorry, I made a small mistake here. `Geth be deployed` should be `Geth can be deployed`. Please update it.